docs/installing/preparing_edge_cluster.md

```diff
@@ -20,24 +20,26 @@ lastupdated: "2021-02-20"
 Perform the following tasks to install an edge cluster and get it ready for the {{site.data.keyword.edge_notm}} agent:
 
 Install one of these edge clusters and prepare it for the {{site.data.keyword.edge_notm}} agent:
-- [Preparing an edge cluster](#preparing-an-edge-cluster)
-  - [Install an OCP edge cluster](#install-an-ocp-edge-cluster)
-  - [Install and configure a k3s edge cluster](#install-and-configure-a-k3s-edge-cluster)
-  - [Install and configure a microk8s edge cluster](#install-and-configure-a-microk8s-edge-cluster)
+- [Preparing an edge cluster](#preparing_an_edge-_luster)
```

Comment:
We may need someone to try to replicate these instructions to confirm that they are still valid.

Reply:
Sounds like a good effort outside the scope of this work.  This commit just syncs the docs and fixes up the anchor tags.